sources/packages/web/src/views/institution/AppInstitution.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <v-app-bar dense flat app>
+  <v-app-bar dense flat app style="overflow:visible">
```

Comment:
can you share, why this style was added?

Reply:
It is also not clear to me also.

Reply:
The same style is used in the nav bar of student page. AppStudent.vue. without this style items in menu bar hide inside the nav bar.

Like this

![image](https://user-images.githubusercontent.com/54600590/129412069-2798af07-ef39-413e-89b7-beb0749c2ad3.png)

Instead of 

![image](https://user-images.githubusercontent.com/54600590/129412129-05c41975-ec52-4fab-9a8d-a695a9a45e1d.png)

Reply:
Yes, now I remembered. It is because we are using PrimeVue because Veutify was not ready during the time that the Student was updated.